src/test/java/junitTest/JunitTest.java

```diff
@@ -0,0 +1,51 @@
+package junitTest;
```

Comment:
Please try to follow code conventions for package names. In Java it should be lowercase - in this case it can be `junittest` or just `junit`, since we are already in test package